apps/ios/Shared/Views/Onboarding/WhatsNewView.swift

```diff
@@ -314,6 +314,42 @@ private let versionDescriptions: [VersionDescription] = [
             ),
         ]
     ),
+    VersionDescription(
+        version: "v5.5",
+        post: URL(string: "https://simplex.chat/blog/20240124-simplex-chat-infrastructure-costs-v5-5-simplex-ux-private-notes-group-history.html"),
+        features: [
+            FeatureDescription(
+                icon: "folder",
+                title: "Private notes",
+                description: "With encrypted files and media."
+            ),
+            FeatureDescription(
+                icon: "link",
+                title: "Simpler interface to connect",
+                description: "Paste the link into the search bar!"
+            ),
+            FeatureDescription(
+                icon: "list.bullet.rectangle",
+                title: "Join group conversations!",
```

Comment:
two exclamation marks in a row
```suggestion
                description: "Paste the link into the search bar!"
            ),
            FeatureDescription(
                icon: "list.bullet.rectangle",
                title: "Join group conversations",
```